deploy.sh

```
@@ -21,15 +21,16 @@ ensure_logs_bucket() {
 TIMESTAMP="$(date -u +'%Y-%m-%dT%H:%M:%S.%NZ')"
 BUILD_LOG="/tmp/mlb-standings-001-build-${TIMESTAMP}.log"
 DEPLOY_LOG="/tmp/mlb-standings-001-deploy-${TIMESTAMP}.log"
-cd $(realpath "$(dirname "${BASH_SOURCE[0]}")")/src &&
+cd $(realpath "$(dirname "${BASH_SOURCE[0]}")") &&
```

Shellcheck (0.10.0)

[warning] Quote this to prevent word splitting.

(SC2046)